lib/response.ex

```diff
@@ -32,11 +69,19 @@ defmodule HTTPServer.Response do
     }[status_code]
   end
 
-  def format_response(res) do
-    "#{res.resource} #{res.status_code} #{res.status_message}" <>
+  def format_response(
+        _res = %__MODULE__{
```

Comment:
I know you are not using res anymore, so you can just drop it completely 

```suggestion
        %__MODULE__{
```

Reply:
Oh wow, for some reason I thought you still had to declare the variable. There are actually quite a few places I can make this change too!!